x-pack/legacy/plugins/siem/public/components/paginated_table/index.tsx

```diff
@@ -227,106 +211,95 @@ export const PaginatedTable = memo<SiemTables>(
         </EuiContextMenuItem>
       ));
     const PaginationWrapper = showMorePagesIndicator ? PaginationEuiFlexItem : EuiFlexItem;
+
     return (
-      <EuiPanel
+      <Panel
         data-test-subj={dataTestSubj}
+        loading={loading}
         onMouseEnter={() => setShowInspect(true)}
         onMouseLeave={() => setShowInspect(false)}
       >
-        <BasicTableContainer>
-          {loading && (
-            <>
-              <BackgroundRefetch />
-              <LoadingPanel
-                height="100%"
-                width="100%"
-                text={`${i18n.LOADING} ${loadingTitle ? loadingTitle : headerTitle}`}
-                position="absolute"
-                zIndex={3}
-                data-test-subj="LoadingPanelPaginatedTable"
-              />
-            </>
-          )}
+        <HeaderPanel
+          id={id}
+          showInspect={!loadingInitial && showInspect}
+          subtitle={
+            !loadingInitial && `${i18n.SHOWING}: ${headerCount.toLocaleString()} ${headerUnit}`
+          }
+          title={headerTitle}
+          tooltip={headerTooltip}
+        >
+          {!loadingInitial && headerSupplement}
+        </HeaderPanel>
+
+        {loadingInitial ? (
+          <EuiLoadingContent data-test-subj="InitialLoadingPanelPaginatedTable" lines={10} />
```

Comment:
nit: Your style usually looks to have `data-test-subj="` start with a lower case letter. Doesn't bother me though, but up to you if you want to keep them consistent.

Reply:
👍